_posts/2024-04-04-chat-with-llms-using-chattr/chat-with-llms-using-chattr.Rmd

```diff
@@ -0,0 +1,310 @@
+---
+title: "Chat with AI in RStudio"
+description: >
+  Interact with Github Copilot, and OpenAI's GPT (ChatGPT) models directly in
```

Comment:
```suggestion
  Interact with Github Copilot and OpenAI's GPT (ChatGPT) models directly in
```
Unbalanced comma, either add a comma after "models" or remove the first comma. I suggest removing.